sh/install-dependencies.sh

```diff
@@ -8,7 +8,7 @@ readonly PYTHON="python3";
 apt-get -y install libxml2-utils mysql-server tree libmysqlclient-dev python3-pip
 ################################################################################
 # 2) INSTALL DISTANT PACKAGE
-python3 -m pip install pytest sphinx wheel setuptools
+python3 -m pip install pytest sphinx wheel setuptools flask singleton_decorator flask-cors
```

Comment:
There are already in the setup.py https://github.com/phe-sto/enthic/blob/master/python/setup.py . The only one required here are the one required by setup.py .